kernel/src/client/default/filesystem.rs

```diff
@@ -6,7 +6,10 @@ use object_store::path::Path;
 use object_store::DynObjectStore;
 use url::Url;
 
-use crate::{executor::TaskExecutor, DeltaResult, Error, FileMeta, FileSlice, FileSystemClient};
+use crate::{
+    client::default::executor::TaskExecutor, DeltaResult, Error, FileMeta, FileSlice,
+    FileSystemClient,
+};
```

Comment:
nit: Another case where two simpler imports seems nicer?
```suggestion
use crate::client::default::executor::TaskExecutor;
use crate::{DeltaResult, Error, FileMeta, FileSlice, FileSystemClient};
```